search-app/src/cmr/search/results_handlers/generics_json_results_handler.clj

```diff
@@ -75,9 +75,9 @@
                         :native_id native-id
                         :name name
                         :id id
-                        :generic-associations (some-> associations-gzip-b64
-                                                      util/gzip-base64->string
-                                                      edn/read-string)})]
+                        :associations (some-> associations-gzip-b64
```

Comment:
this is technically a breaking change, is there any other notice of this that needs to be made, say in a CHANGES.md file?

Reply:
true, but no one has used this.  The generic capabilities have been turned off in UAT and is not yet in PROD.  I won't be turned on until these changes are in production.